zombienet_tests/functional/0002-parachains-disputes.toml

```
@@ -0,0 +1,71 @@
+[settings]
+timeout = 1000
+
+[relaychain.genesis.runtime.runtime_genesis_config.configuration.config]
+  max_validators_per_core = 2
+  needed_approvals = 2
+
+[relaychain]
+default_image = "{{ZOMBIENET_INTEGRATION_TEST_IMAGE}}"
+chain = "rococo-local"
+chain_spec_command = "polkadot build-spec --chain rococo-local --disable-default-bootnode"
+default_command = "polkadot"
+  
+  [[relaychain.nodes]]
+  image = "{{MALUS_IMAGE}}"
+  name = "alice"
+  command = "malus dispute-ancestor"
+  extra_args = [ "--alice", "-lparachain=debug" ]
+
+  [[relaychain.nodes]]
+  image = "{{MALUS_IMAGE}}"
+  name = "bob"
+  default_command = "malus dispute-ancestor"
```

```suggestion
  command = "malus dispute-ancestor"
```

This should be command.